src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -661,6 +661,13 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
 
         // Discover translation directories
         $dirs = array();
+
+        foreach ($config['paths'] as $path) {
+            if (is_dir($path)) {
+                $dirs[] = $path;
+            }
```

Comment:
Is it really a good idea to silently ignore values not representing an array? This would make debugging a bit harder I guess. What about doing validation in the `Configuration` class instead and providing a meaningful error message?